hadoop-ozone/dist/src/main/smoketest/commonlib.robot

```diff
@@ -31,6 +31,16 @@ Get test user principal
     ${instance} =       Execute                    hostname | sed 's/scm[0-9].org/scm/;s/scm[0-9]/scm/;s/om[0-9]/om/'
     [return]            ${user}/${instance}@EXAMPLE.COM
 
+Get Security Enabled From Config
+    ${conf_dir} =    Get Environment Variable    OZONE_CONF_DIR
+    ${file} =     Set Variable    ${conf_dir}/ozone-site.xml
+    ${value} =    Execute    sed -n "/<name>ozone.security.enabled<\\/name>/,/<\\/property>/s|.*<value>\\(.*\\)</value>.*|\\1|p" ${file} | head -n1 | xargs
+    IF    '${value}' != 'true' and '${value}' != 'false'
```

Comment:
I have not tested it, but just my doubt, that this is not string literal comparison (not variable). If ${value} is actually being evaluated ?

Reply:
`'${value}'` looks fine, we use this syntax elsewhere.  `''` ensures we don't get syntax errors if value is empty.